src/buckets.spec.ts

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable import/first */
```

Comment:
These tests are all pretty much exactly the same as the go versions. The only difference here is I only create one account and one bucket and keep using that one along the way. Note that there are tests specific to browser and node. This is a good flow to use for our other packages that need to run different code in different envs as well.